### ISSUE TITLE
Should it be `BlockedStatus`?

### ISSUE THREAD

**Issue by Abuelodelanada** [canonical/grafana-agent-k8s-operator#104]
I'm wondering whether we should even have this block? We don't do this with loki_push_api right?

Also, if we keep it, it should be `BlockedStatus`.

_Originally posted by @dstathis in https://github.com/canonical/grafana-agent-k8s-operator/pull/95#discussion_r1127686471_

**Comment by rbarry82** [comment]
We don't do this with `loki_push_api`, but Loki pushes batch, and wait in the queue until it actually is available (so in theory, waiting for a bit doesn't actually cause lost logging data, just delayed logging data). AFAIK, remote-write does not, and may cause Prometheus to complain about out of order timestamps even if it did, so waiting is not ok

**Comment by sed-i** [comment]
According to both docstrings and the doc page, "waiting" means the charm is healthy, and "blocked" - unhealthy.

I think "waiting" is correct here.

---

[Doc page](https://juju.is/docs/olm/status-values):

Status | Meaning
-- | --
waiting | The charm is alive and well, and is waiting for some event to occur. No human intervention required.
blocked | The charm is stuck. Human attention/intervention is required.


Docstring for [WaitingStatus](https://github.com/canonical/operator/blob/eb8e25a5314ca2063f4689fa829c8bdca652ce1f/ops/model.py#L1635):
```
    The unit is unable to progress to an active state because an application to which
    it is related is not running.
```

Docstring for [BlockedStatus](https://github.com/canonical/operator/blob/eb8e25a5314ca2063f4689fa829c8bdca652ce1f/ops/model.py#L1614):
```
    An operator has to manually intervene to unblock the unit and let it proceed.
```

**Comment by sed-i** [comment]
Fixed in #141.
